listenbrainz/spotify_updater/spotify_read_listens.py

```diff
@@ -246,9 +248,8 @@ def parse_and_validate_spotify_plays(plays, listen_type):
         try:
             validate_listen(listen, listen_type)
             listens.append(listen)
-        except BadRequest as e:
-            current_app.logger.error(str(e))
-            raise
+        except APIBadRequest as e:
+            current_app.logger.info(str(e))
```

Comment:
this still logs the error and the raise is discarded. Should it be the other way round?

Reply:
Its intended to be this way. We do not want to report APIBadRequest to sentry hence the logging level. I removed the raise because we want to ignore invalid listens. 

We do so because these listens are received from Spotify not users. As discussed in IRC, removed the logging as well.